### ISSUE TITLE
new parameter: QoS

### ISSUE THREAD

**Issue by cubilon** [MacWyznawca/homebridge-mqtt-switch-tasmota#42]
just a little parameter addon: QoS for MQTT.
please 🙏

**Comment by mu2004** [comment]
I agree it's a useful parameter to ensure a failed WIFI connection doesn't continually toggle the switch's on off state.

**Comment by peros550** [comment]
Tasmota supports QoS on its side?
